### DISCUSSION TITLE
A simple web/rest API frontend for EdgeGPT

### DISCUSSION THREAD

**Discussion by Mic92** [acheong08/EdgeGPT#334] [Show and tell]
I just quickly hacked something together to extract images/text completions.
The ui is quite rudimentary as my main use case is the API.

https://github.com/Mic92/bing-gpt-server

**Comment by Nu11Const** [comment]
Can you support the stream?

**Reply by Mic92** [reply]
No. This is just a basic REST API which we use for IRC bots and where we wanted to avoid the complexity of websockets. Users who want that might just use your https://github.com/VMTask/NewBing-Web instead.
